programs/clearing_house/src/controller/orders.rs

```diff
@@ -1013,7 +1017,19 @@ pub fn fulfill_order_with_amm(
             order_post_only,
         )?;
 
+    let market_postion_unsettled_pnl_delta = cast_to_i128(user_fee)?
```

Comment:
@0xbigz

Reply:
this is for the quote collected by the amm position and position_per_lp. it excludes the quote_asset_surplus since thats already baked into the price at exercise